android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -43,7 +42,7 @@ class TranslationsMiddleware(
         // Pre process actions
         when (action) {
             is TranslateExpectedAction -> {
-                scope.launch {
+                scope.launch(Dispatchers.Main) {
```

Comment:
Since this scope already has its dispatcher set to Main in its context [here](https://searchfox.org/mozilla-mobile/rev/c3447daa62d50e8b05251a9610ea9611b66705c8/firefox-android/android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/EngineMiddleware.kt#39), you shouldn't need to specify it explicitly

Reply:
Updated, also thanks for the conversation on Slack!